.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+# Description
+<!-- Explain here the changes your PR introduces and text to help us understand the context of this change. -->
+
+# Contribution Checklist:
+- [ ] **The pull request does not introduce any breaking changes**
+- [ ] **I have read the [contribution guidelines](https://github.com/oohsai/rust/blob/master/CONTRIBUTING.md).**
+- [ ] **Create an issue and link to the pull request.**
+
+## Attach Screenshots (Optional) 
+
+
```

Comment:
of what?
code? compiler output? other information that we already have present in textual form?

Reply:
I am just a beginner. As HacktoberFest started, I was just trying to contribute to make the PR Request more Interactive. I understand your Feedback. Thank You for picking out the error.

Reply:
> I am just a beginner. As HacktoberFest started, I was just trying to contribute to make the PR Request more Interactive. I understand your Feedback. Thank You for picking out the error.

This repo is not marked as »Hacktoberfest« , so it will not count towards Hacktoberfest. (It could still count if »Hacktoberfest-Accepted« label is applied to this pr, but I don't think the rust team wants to go down that route). 

If you are doing this just for hacktoberfest, I would consider finding a repo where your contribution will be accepted for the competition. 

If you are doing this to contribute to rust, you are welcome to continue with this